okio/jvm/jmh/src/jmh/java/com/squareup/okio/benchmarks/HashFunctionBenchmark.java

```diff
@@ -43,7 +43,7 @@
 public class HashFunctionBenchmark {
 
   MessageDigest jvm;
-  HashFunction okio;
+//  HashFunction okio;
```

Comment:
Is this because you made it `internal`? We can use friend paths, but it doesn't have IDE support yet.

Reply:
Can you link a reference on that? I'm not seeing anything stand out when I search for "Kotlin friend path". I haven't heard of them before.

Reply:
https://youtrack.jetbrains.com/issue/KT-20760